whatsagent/serializers.py

```diff
@@ -0,0 +1,12 @@
+from rest_framework import serializers
+
+
+class SignupSerializer(serializers.Serializer):
```

Comment:
- Create `whatsagent/api/serializers` Python package
- Create `signup.py` in `whatsagent/api/serializers` Python package
- Move `SignupSerializer` to `whatsagent/api/serializers/signup.py`

Reply:
serializers now under folder api